projects/packages/search/src/class-options.php

```diff
@@ -48,6 +48,7 @@ class Options {
 	const OVERLAY_TRIGGER_IMMEDIATE = 'immediate';
 	const OVERLAY_TRIGGER_RESULTS   = 'results';
 	const OVERLAY_TRIGGER_SUBMIT    = 'submit';
+	const DEFAULT_OVERLAY_TRIGGER   = 'submit';
```

Comment:
```suggestion
	const DEFAULT_OVERLAY_TRIGGER   = OVERLAY_TRIGGER_SUBMIT;
```
Would it be better to assign DEFAULT_OVERLAY_TRIGGER to the default trigger rather than be given a string?

Reply:
You're right @dognose24! For some reason I thought you couldn't use the content of another class constant. It looks like it'd have to use `self` though:

`const DEFAULT_OVERLAY_TRIGGER = self::OVERLAY_TRIGGER_SUBMIT;`

Reply:
You are right! I just tested the code in a pure PHP playground. To access constant in Class should use `self`.

Reply:
Fixed :)